src/openms/source/FORMAT/OSWFile.cpp

```diff
@@ -46,28 +46,22 @@ namespace OpenMS
   namespace Sql = Internal::SqliteHelper;
   using namespace std;
 
-  OSWFile::OSWFile()
-  {
-  }
-
-  OSWFile::~OSWFile()
-  {
-  }
+  const std::array<std::string, (Size)OSWFile::OSWLevel::SIZE_OF_OSWLEVEL> OSWFile::names_of_oswlevel = { "ms1", "ms2", "transition" };
```

Comment:
would uppercase be better? MS1, MS2, transition

Reply:
this would break existing workflows.
I dont mind either way.
Preferences?